tests/models/test_db.py

```diff
@@ -0,0 +1,7 @@
+from headjack.models import db
+from headjack.models.tool import ToolSchema
+
+def test_saving_a_tool_schema(session):
```

Comment:
Simple example of saving a tool to the metadata db and then retrieving it back.

Reply:
@samredai let's merge this! we can build out more tests once this design with schemas is better validated.